autogluon/utils/tabular/ml/models/lr/lr_preprocessing_utils.py

```diff
@@ -0,0 +1,74 @@
+from scipy.sparse import hstack
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.preprocessing import OneHotEncoder
+
+
+def get_one_hot_features(df):
+    ohe_cats = df.nunique()
+    ohe_cats = ohe_cats[ohe_cats < 50].index.values.tolist()
```

Comment:
should 50 be a tracked argument here? perhaps eventually something to search over?

Reply:
This method was removed - remainings of early prototype code.